plugin/storage/grpc/README.md

```diff
@@ -62,6 +62,8 @@ A plugin must implement the StoragePlugin interface of:
 type StoragePlugin interface {
    	SpanReader() spanstore.Reader
    	SpanWriter() spanstore.Writer
+	ArchiveSpanReader() shared.ArchiveReader
+	ArchiveSpanWriter() shared.ArchiveWriter
```

Comment:
Extending the interface won't be backwards compatible, contrary to the PR description. Perhaps it would be better to introduce an optional interface 
```
type ArchiveStoragePlugin interface {
   	ArchiveSpanReader() spanstore.Reader
   	ArchiveSpanWriter() spanstore.Writer
}
```
If the plugin does not implement this interface then the corresponding methods on the grpc client (which is internal to Jaeger) can return an error. It is the same pattern as the used by the main `storage.Factory` and `storage.ArchiveFactory`.

Reply:
This doesn't match now.

Reply:
Can we do another interface for Archive?